docs/user/recipes/msgspec-integration.rst

```diff
@@ -90,17 +89,20 @@ case is represented by ``msgspec.ValidationError``. We could either create an
 exception as :class:`~falcon.MediaValidationError`, or just use a
 ``try.. except`` clause, and reraise directly inside middleware.
 
-Our customized :class:`~falcon.media.JSONHandler` has another issue: unlike the
-stdlib's :mod:`json` or the majority of other JSON libraries,
-``msgspec.DecodeError`` is not a subclass of :class:`ValueError`:
+``msgspec.json.decode()`` has another peculiarity: unlike the stdlib's
+:mod:`json` or the majority of other JSON libraries, ``msgspec.DecodeError`` is
+not a subclass of :class:`ValueError`:
 
 >>> import msgspec
 >>> issubclass(msgspec.DecodeError, ValueError)
 False
 
-This discrepancy can be worked around using a wrapper around
-``msgspec.json.decode()`` that reraises a :class:`ValueError` from
-``msgspec.DecodeError``. Furthermore, the problem has been reported upstream,
+Falcon's :class:`~falcon.media.JSONHandler` works around this discrepancy by
+detecting the actual deserialization exception type at the time of
+initialization, but you may still encounter the issue when using the library
+manually, or when using it for other media types such as MessagePack
+(see above).
+Furthermore, the problem has been reported upstream,
```

Comment:
oh ok, you have a link?

Reply:
Yes, reported by me :slightly_smiling_face: 
https://github.com/jcrist/msgspec/issues/634